docs/decisions/0001-multiple-enterprise-support.rst

```diff
@@ -0,0 +1,47 @@
+Multiple Enterprise Support
+---------------------------
+
+Status
+======
+
+Draft
+
+Context
+=======
+
+There is a business need to be able to support a learner being linked to multiple enterprises. Today, edX workflows only support a 1-to-1 linkage and our system does not prevent linking multiple Enterprises to a single learner. Hence not all workflows work as expected.
+
+ Some examples of these errant workflows are:
+
+#. When a learner is linked with an enterprise and tries to link with another enterprise(through SSO), the learner is redirected to the login page instead of the registration page for the second enterprise. This is because the system detects that these credentials are already in the system.
+#. Enrollment, Progression, Completion, Consent records become ambiguous if a learner completes a course from enterprise A and then attempts the same course through enterprise B.
+#. Incorrect discounts are applied when an employee linked with multiple enterprises redeems the benefit applicable only to one of the enterprises.
+
+
+Decisions
+=========
+
+Short term:
+#. As part of the login flow the user would select one of the linked enterprises.
```

Comment:
Can we some information regarding `What steps a logged in learner has to do to switch to another enterprise?`

Reply:
Some information is added in the "Long term" section. More detail would be added as part of the detailed solution.